OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignalCacheCleaner.java

```diff
@@ -1,24 +1,19 @@
 package com.onesignal;
 
-import com.onesignal.OneSignalDbContract.NotificationTable;
-import com.onesignal.influence.model.OSInfluenceChannel;
-import com.onesignal.outcomes.OSOutcomeTableProvider;
-
 import android.content.Context;
-import android.database.Cursor;
+import android.database.SQLException;
 import android.database.sqlite.SQLiteDatabase;
 import android.os.Process;
 import android.support.annotation.WorkerThread;
 
-import org.json.JSONArray;
-import org.json.JSONException;
-
-import java.util.Set;
+import com.onesignal.OneSignalDbContract.NotificationTable;
+import com.onesignal.influence.model.OSInfluenceChannel;
+import com.onesignal.outcomes.OSOutcomeTableProvider;
 
 class OneSignalCacheCleaner {
 
     private final static long NOTIFICATION_CACHE_DATA_LIFETIME = 604_800L; // 7 days in seconds
-    private final static long IAM_CACHE_DATA_LIFETIME = 15_552_000L; // 6 months in seconds
+    final static long IAM_CACHE_DATA_LIFETIME = 15_552_000L; // 6 months in seconds
```

Comment:
maybe move it to the IAM Repository file?

Reply:
Moved to IAM Repository file